parse_test.go

```diff
@@ -845,6 +978,17 @@ func TestEnvironmentVariableInSubcommandIgnored(t *testing.T) {
 	assert.Equal(t, "", args.Sub.Foo)
 }
 
+func TestNonPointSubcommand(t *testing.T) {
```

Comment:
`TestNonPointerSubcommand`

Reply:
Yup. Fixed. Found and fixed a new more like that.